### DISCUSSION TITLE
zh/%E5%A4%96%E8%AF%AD%E5%AD%A6%E4%B9%A0/%E6%97%A5%E6%9C%AC%E8%AF%AD/side_notes_%E6%AF%94%E8%BE%83%EF%BC%91

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#555] [Announcements]
# zh/%E5%A4%96%E8%AF%AD%E5%AD%A6%E4%B9%A0/%E6%97%A5%E6%9C%AC%E8%AF%AD/side_notes_%E6%AF%94%E8%BE%83%EF%BC%91

「より」、「以上に」、「一方」都有比较和对比的意思。 「より」 提示 最为常用。也有「よりも」的用法，比较意味更强。 构成方法与接续方法 名词 + 「より」 例： 私は昨日よりも今日の方が元気です.

https://blog.solex-inc.com/zh/%E5%A4%96%E8%AF%AD%E5%AD%A6%E4%B9%A0/%E6%97%A5%E6%9C%AC%E8%AF%AD/side_notes_%E6%AF%94%E8%BE%83%EF%BC%91.html

**Comment by 100vision** [comment]
另，新标日/中级上册/第16课中， `「　~のに対して　」` 也有对比，比较的意思

**Reply by 100vision** [reply]
例えば、

「彼女の仕事は簡単だった。しかし、彼の仕事の場合のに対して、非常に複雑だった。」
(Her job was easy. However, his job was very complicated in comparison.)

**Comment by Mister-Hope** [comment]
https://github.com/vuepress-theme-hope/giscus-discussions/blob/main/README.md
